frontend/lib/src/components/shared/StreamlitMarkdown/styled-components.ts

```diff
@@ -71,6 +71,9 @@ export const StyledStreamlitMarkdown =
         },
 
         li: {
+          // TODO(lukasmasuch): We might want to refactor
```

Comment:
If not already happened, it might make sense to add a tech debt item for this as well

Reply:
I'm added it to this tech debt item: https://www.notion.so/snowflake-corp/Clean-up-remaining-hardcoded-CSS-values-11a7170bb416806190f3d3f48196e177